Makefile.validation

```diff
@@ -147,3 +147,33 @@ kind-create-cluster:
 
 kind-delete-cluster:
 	$(KIND) delete cluster
+
+sidecars-pull:
```

Comment:
Not all images are sidecars, correct? Can we rename to `deps-docker-pull`?

Reply:
renamed